### PR TITLE
chore: activate input compression for encrypted data-frames [BLOCKED BY CP]

### PR DESCRIPTION
**Pull request by RomanBredehoft**
We realized with @andrei-stoian-zama that encrypted data-frames did not have the input compression feature activated 

this is currently blocked by https://github.com/zama-ai/concrete-internal/issues/758

### CODE REVIEW

File: src/concrete/ml/sklearn/linear_model.py [hunk 1]
```diff
@@ -347,7 +347,7 @@ def _get_training_quantized_module(
         # Enable the underlying FHE circuit to be composed with itself
         # This feature is used in order to be able to iterate in the clear n times without having
         # to encrypt/decrypt the weight/bias values between each loop
-        configuration = Configuration(composable=True, compress_evaluation_keys=True)
```
Comment (RomanBredehoft) [inline src/concrete/ml/sklearn/linear_model.py:350]: the feature is activated in the QuantizedModule's compile function, so let's avoid some confusion here

File: tests/torch/test_hybrid_converter.py [hunk 2]
```diff
@@ -37,17 +37,9 @@ def run_hybrid_llm_test(
 ):
     """Run the test for any model with its private module names."""
 
-    # Multi-parameter strategy is used in order to speed-up the FHE executions
-    configuration = Configuration(
```
Comment (RomanBredehoft) [inline tests/torch/test_hybrid_converter.py:41]: same, these options are the QM's default, so let's avoid some confusion / unexpected mismatches

**Review by andrei-stoian-zama — APPROVED** [review]
Thanks!

**Comment by andrei-stoian-zama** [comment]
will be added in #805

**Pull request closed without merging**